### PR TITLE
Tre new layout

### PR DESCRIPTION
**Pull request by JsseL**
# WIP*: Merge upstream v0.2.0 and add Tampere customizations

Merge upstream changes up until v0.2.0 release to `tre` branch which is the active Tampere specific development branch.

On top of upstream changes, small Tampere specific customization work is done to meet the Tampere brand requirements.

Merging this will also be the end of the old map based UI that Tampere has used and a move to the same UI that Helsinki and other cities use.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "presets": ["es2015", "react", "stage-2"]
-}
+  "presets": ["@babel/preset-env", "@babel/preset-react"],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties"
+  ],
+}
```
Comment (suutari-ai) [inline .babelrc]: End of line missing at the end of file

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,4 @@
+CLIENT_ID=
+CLIENT_SECRET=
+SESSION_SECRET=
+TARGET_APP=
```
Comment (suutari-ai) [inline .env.example]: End of line missing at the end of file

File: .eslintrc [hunk 3]
```diff
@@ -63,7 +61,30 @@
...
+    "jsx-a11y/no-static-element-interactions": "off",
+    
+    "jsx-a11y/no-noninteractive-tabindex": "off",
+    
+    "react/no-array-index-key": "off",
 
+    "comma-dangle": "off"
   }
-}
+}
```
Comment (suutari-ai) [inline .eslintrc]: eol missing

**Review by suutari-ai — APPROVED** [review]
Can't really review anything this big, but I skimmed it through and checked the topmost commits and trust it's OK.

File: app/pages/home/_home-page.scss [hunk 4]
```diff
@@ -53,28 +55,55 @@
       margin-top: 0;
     }
 
-    &__banners {
-      display: flex;
-      flex-direction: row;
+    // &__banners {
+    //   display: flex;
+    //   flex-direction: row;
```
Comment (jopesy) [inline app/pages/home/_home-page.scss]: Maybe just remove these instead of commenting them out?

File: app/pages/home/_home-page.scss [hunk 5]
```diff
@@ -92,19 +121,19 @@
...
+      //   margin: 30px 0;
+      // }
+      // h5 {
+      //   margin: 12px 12px 90px 12px;
+      // }
+      // &-action {
+      //   position: absolute;
+      //   bottom: 30px;
+      //   width: 100%;
+      // }
```
Comment (jopesy) [inline app/pages/home/_home-page.scss]: Same here, just remove the obsolete lines

**Comment by suutari-ai** [comment]
Duplicate of #4 and these changes are already integrated to `tre` branch in #6. Thanks @JsseL and @blomqma! :+1:

**Pull request closed without merging**